specs/_features/eip7594/polynomial-commitments-sampling.md

```diff
@@ -671,27 +671,22 @@ def verify_cell_kzg_proof(commitment_bytes: Bytes48,
 #### `verify_cell_kzg_proof_batch`
 
 ```python
-def verify_cell_kzg_proof_batch(row_commitments_bytes: Sequence[Bytes48],
-                                row_indices: Sequence[RowIndex],
+def verify_cell_kzg_proof_batch(commitments_bytes: Sequence[Bytes48],
                                 column_indices: Sequence[ColumnIndex],
```

Comment:
```suggestion
                                cell_indices: Sequence[ColumnIndex],
```

Reply:
I think its better to remove the column_indices terminology since the cryptography does not know what a column is

Reply:
Yes definitely. I'd like to do this in a follow up PR though, to keep things relatively simple.